packages/signature-controller/src/SignatureController.ts

```diff
@@ -364,9 +373,16 @@ export class SignatureController extends BaseControllerV2<
     version: string,
     signingOpts: TypedMessageSigningOptions,
   ): Promise<string> {
+    let signTypeForLogger = SigningMethod.EthSignTypedData;
```

Comment:
Really minor, but is it worth using a nested ternary here or a tiny helper function?

Reply:
I've made it on helper function, nested ternary makes TS to complain.
Fixed in https://github.com/MetaMask/core/pull/1692/commits/95219fbdd86483e0a5830fb89e6de7f367b47b8b